ui/page/governance/proposal_details_page.go

```diff
@@ -523,7 +523,7 @@ func (pg *ProposalDetails) Layout(gtx C) D {
 				var err error
 				proposalDescription, err = pg.WL.MultiWallet.Politeia.FetchProposalDescription(proposal.Token)
 				if err != nil {
-					fmt.Printf("Error loading proposal description: %v", err)
+					log.Errorf("Error loading proposal description: %v", err)
```

Comment:
seems like all error formatting are intentionally left localized, can you shed more light?

Reply:
okay seem like we want all the logs in english, disregard